Refresh.Interfaces.Game/Endpoints/Levels/PublishEndpoints.cs

```diff
@@ -211,6 +213,23 @@ public Response PublishLevel(RequestContext context,
                 return BadRequest;
             }
         }
+        
+        // Automatically mark level as reupload by keyword matching the title
+        // + get original publisher from the description??
+        string[] reuploadWords = { 
+            "(reupload)", "[reupload]",
+            "(archive)", "[archive]"
+        };
```

Comment:
This should be stored in the class (perhaps statically) as this causes an allocation every upload.